config/app.php

```diff
@@ -40,8 +40,8 @@
     | or doing any other checks to ensure the service is functional.
     |
     */
-    
-    'preflight' => false,
+
+    'preflight' => true,
```

Comment:
As a compromise, I would say to have preflight set to WP_ENV !== 'production' or something so that by default it's not enabled in production.

Reply:
That works.

Reply:
I guess one afterthought I have on this is the default location of the `cache` being in `uploads` – perhaps this might be problematic for those who are just pushing updates to their base theme and not pushing the new cache up?

If so, would putting `cache` in `resources` maybe be more sane?